README.md

```diff
@@ -14,8 +18,6 @@ To edit files, proceed to edit index.html and index.css
 
 ## Authors
```

Comment:
- [ ] Good job here! Your readme is very descriptive but it needs a few tweaks and you're good to go!
- A good README should have a description that is clear, short, and to the point. It should describe the importance of your project, what it does and how to use it. This way technical and non-technical users can understand how to use it.
- Include acknowledgements to Figma for the design template.
- The `Getting Started` section which includes information on how to run the project locally needs to be updated and include the necessary steps to clone it, noting that there might be non-technical users as well.
- Check out how to use [markdowns](https://www.markdownguide.org/cheat-sheet/) for hyperlinks in the readme.
- Also, you have a horizontal overflow in your project on Desktop screens, kindly fix it.